lib/platform/git/storage.ts

```diff
@@ -1,15 +1,33 @@
-const fs = require('fs-extra');
-const { join } = require('path');
-const path = require('path');
-const URL = require('url');
-const Git = require('simple-git/promise');
-const convertHrtime = require('convert-hrtime');
+/// <reference path="../../../node_modules/simple-git/promise.d.ts" />
+
+import fs from 'fs-extra';
+import { join } from 'path';
+import path from 'path';
+import URL from 'url';
+import Git from 'simple-git/promise';
+import convertHrtime from 'convert-hrtime';
+
+declare module 'fs-extra' {
+  export function exists(pathLike: string): Promise<boolean>;
```

Comment:
Is this necessary? Doesn't `fs-extra` have its own typings?

Reply:
Yes, but this overload is missing